app/controllers/lottery_controller.rb

```diff
@@ -74,7 +74,7 @@ def create_new_teams_for_bidding_response(teams, assignment)
   # to strongest contenders (team strength, priority of bids)
   def run_intelligent_bid
     unless Assignment.find_by(id: params[:id]).is_intelligent # if the assignment is intelligent then redirect to the tree display list
-      flash[:error] = "This action not allowed. The assignment " + Assignment.find_by(id: params[:id]).name + " does not enabled intelligent assignments."
+      flash[:error] = "This action is not allowed. The assignment " + Assignment.find_by(id: params[:id]).name + " does not enable intelligent assignments."
```

Comment:
Thank you for those fixes. We will merge this pull request. Just remember to do a git pull b4 you continue.